### ISSUE TITLE
🐛 Mobskills refering to tryInterruptSpell that does not exist

### ISSUE THREAD

**Issue by bope12** [DerpyProjectGroup/topaz#2171]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)

## Branch affected by issue 

`base`

## Steps to reproduce
Have a mob use one of the following mobskills
Dancing Edge, Evisceration, Impulse Drive, Penta Thrust, Shadowstitch, Skewer, Spiral Hell, Vorpal Scythe, Wheeling Thrust
at the end of all of them it tries to call tryInterruptSpell but no such luaentity binding exists
## Expected behavior

Not to error out on skill usage

**Comment by EpicTaru** [comment]
Looks like it may be a typo issue, I'll test and pr a fix after I return from lunch unless someone else does it before then.

**Comment by EpicTaru** [comment]
Actually, I'm not able to recreate this. Do you have a capture of that warning popping up in the console, @bope12?

**Comment by claywar** [comment]
Mobskills for the above items call `target:tryInterruptSpell(mob, info.hitslanded)` for which no binding exists.  Looks like the intent is to make it to the equivalent battleutils function.

**Comment by EpicTaru** [comment]
Correct, but I see no warning or error at all in the map console when a mob uses one of the listed mobskills.

**Comment by bope12** [comment]
> Actually, I'm not able to recreate this. Do you have a capture of that warning popping up in the console, @bope12?

The log shows just exactly what I put, if you are not getting that error are you on a nonstock repo that has that method added?
`[06/26/22 19:32:20:884][map][error][error] luautils::onMobWeaponSkill (mobskill) ./scripts/globals/mobskills/wheeling_thrust.lua:30: attempt to call method 'tryInterruptSpell' (a nil value)
stack traceback:
	./scripts/globals/mobskills/wheeling_thrust.lua:30: in function <./scripts/globals/mobskills/wheeling_thrust.lua:17> (luautils::OnMobWeaponSkill:3553)`
is adding interrupt in the mobskill even needed or is that handled else where in the damage process
